pkg/collect/logs.go

```diff
@@ -286,6 +286,12 @@ func setLogLimits(podLogOpts *corev1.PodLogOptions, limits *troubleshootv1beta2.
 	} else {
 		podLogOpts.TailLines = &limits.MaxLines
 	}
+
+	if limits.MaxBytes == 0 {
+		podLogOpts.LimitBytes = &limits.MaxBytes
+	} else {
+		podLogOpts.LimitBytes = &limits.MaxBytes
+	}
 }
```

Comment:
what does this conditional do?

Reply:
This was a temporary workaround during testing..This has been resolved.